### PR TITLE
Add Shopify

### PR DESCRIPTION
**Pull request by amatzen**
https://github.com/julionc/awesome-shopify
Shopify is one of the most used ecommerce platform, and is quite a lot extensible. So I think it should be on the official awesome list.

### CODE REVIEW

**Comment by sindresorhus** [comment]
It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

Also try running https://github.com/sindresorhus/awesome-lint on the list.

// @julionc

**Comment by uraimo** [comment]
Hmm, the linter has found 50 errors.

```
  readme.md:83:3
  ✖    1:1    Missing or invalid Table of Contents                                 awesome/toc
  ✖    1:20   Invalid badge source                                                 awesome/badge
  ✖   11:3    Invalid list item link URL                                           awesome/list-item
  ✖   12:3    Invalid list item link URL                                           awesome/list-item
  ✖   13:3    Invalid list item link URL                                           awesome/list-item
  ✖   14:3    Invalid list item link URL                                           awesome/list-item
  ✖   15:3    Invalid list item link URL                                           awesome/list-item
  ✖   16:5    Invalid list item link URL                                           awesome/list-item
  ✖   17:5    Invalid list item link URL                                           awesome/list-item
  ✖   18:5    Invalid list item link URL                                           awesome/list-item
  ✖   18:6    Text "Javascript" should be written as "JavaScript"                  awesome/spell-check
  ✖   19:5    Invalid list item link URL                                           awesome/list-item
  ✖   20:5    Invalid list item link URL                                           awesome/list-item
  ✖   21:5    Invalid list item link URL                                           awesome/list-item
  ✖   22:5    Invalid list item link URL                                           awesome/list-item
  ✖   23:5    Invalid list item link URL                                           awesome/list-item
  ✖   24:5    Invalid list item link URL                                           awesome/list-item
  ✖   25:5    Invalid list item link URL                                           awesome/list-item
  ✖   26:3    Invalid list item link URL                                           awesome/list-item
  ✖   27:5    Invalid list item link URL                                           awesome/list-item
  ✖   28:5    Invalid list item link URL                                           awesome/list-item
  ✖   29:3    Invalid list item link URL                                           awesome/list-item
  ✖   30:3    Invalid list item link URL                                           awesome/list-item
  ✖   38:3    Link to https://www.reddit.com/r/shopify is dead                     no-dead-urls
  ✖   52:160  Text "github" should be written as "GitHub"                          awesome/spell-check
  ✖   52:203  List item description must end with proper punctuation               awesome/list-item
  ✖   66:32   Text "Javascript" should be written as "JavaScript"                  awesome/spell-check
  ✖   69:50   List item link and description must be separated with a dash         awesome/list-item
  ✖   70:70   List item link and description must be separated with a dash         awesome/list-item
  ✖   78:3    Link to https://apps.shopify.com/shopify-mobile-for-android is dead  no-dead-urls
  ✖   83:3    Link to https://apps.shopify.com/shopify-mobile is dead              no-dead-urls
  ✖   97:80   List item description must end with proper punctuation               awesome/list-item
  ✖  101:70   List item description must end with proper punctuation               awesome/list-item
  ✖  108:134  List item description must end with proper punctuation               awesome/list-item
  ✖  111:70   List item description must end with proper punctuation               awesome/list-item
  ✖  113:54   List item description must end with proper punctuation               awesome/list-item
  ✖  121:74   List item description must end with proper punctuation               awesome/list-item
  ✖  122:76   List item description must end with proper punctuation               awesome/list-item
  ✖  127:72   List item description must end with proper punctuation               awesome/list-item
  ✖  141:72   List item description must end with proper punctuation               awesome/list-item
  ✖  153:70   List item description must end with proper punctuation               awesome/list-item
  ✖  154:70   List item description must end with proper punctuation               awesome/list-item
  ✖  155:74   List item description must end with proper punctuation               awesome/list-item
  ✖  156:91   List item description must end with proper punctuation               awesome/list-item
  ✖  157:74   List item description must end with proper punctuation               awesome/list-item
  ✖  161:55   List item description must end with proper punctuation               awesome/list-item
  ✖  170:3    Link to https://github.com/urbandictionary/shopify-printful is dead  no-dead-urls
  ✖  175:3    Link to https://github.com/christianblais/parcelify is dead          no-dead-urls
  ✖  179:66   List item description must end with proper punctuation               awesome/list-item
  ✖  194:325  List item description must end with proper punctuation               awesome/list-item

  50 errors
```

**Comment by julionc** [comment]
I will take a look.
Thanks for advise!

**Comment by julionc** [comment]
I have made changes according to your suggestions (guidelines doc). I hope this help.

Thanks @amatzen to include the Shopify List into this project.
Can you update this PR?  Don't forget put #readme append to the link.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/julionc/awesome-shopify
✖ Linting

  readme.md:169:80
  ✖  169:80  List item description must end with proper punctuation  remark-lint:awesome/list-item

  1 error
```

File: readme.md [hunk 1]
```diff
@@ -580,6 +580,7 @@
 - [Events in the Netherlands](https://github.com/awkward/awesome-netherlands-events#readme) - Tech-related events in the Netherlands.
 - [Healthcare](https://github.com/kakoni/awesome-healthcare#readme) - Open source healthcare software for facilities, providers, developers, policy experts, and researchers.
 - [Magento 2](https://github.com/DavidLambauer/awesome-magento2#readme) - Open Source eCommerce built with PHP.
+- [Shopify](https://github.com/julionc/awesome-shopify) - Commercial and customizable eCommerce platform built on Ruby.
```
Comment (sindresorhus) [inline readme.md]: I don't think the Ruby part is relevant since it's not open source?

**Comment by sindresorhus** [comment]
> The entry in the Awesome list should:
Be added at the bottom of the appropriate category.

**Comment by sindresorhus** [comment]
A few entries are missing a description.

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example)

https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**